ang/civicase/ActivityCardMenu.html

```diff
@@ -4,7 +4,7 @@
   </button>
   <ul class="dropdown-menu dropdown-menu-right">
     <li>
-      <a ng-href="#{{ activityFeedUrl(activity.case_id, null, null, activity.id) }}">
+      <a ng-href="#{{ activityFeedUrl(activity.case_id, null, null, null, activity.id) }}">
```

Comment:
@deb1990 can you please explain this change? It's not clear why an extra `null` was needed and the commit message only reads _"Make quick links hyperlink"_.

Regardless, functions with 3 parameters are hard to read, with 5 it's impossible and can lead to a lot of confusion. I'm guessing this confusion was the reason for the missing `null`. Can we at some point change this function so it uses a single object parameter instead of 5 nameless parameters?

Reply:
Sure @reneolivo . I will add it to the description.
About using a single object, instead of multiple params, I completely agree with that. I will implement this in a future PR for sure. Current ticket is going on for some days so I wanted to get this merged. I hope that is fine :)